lib/routes/account.js

```diff
@@ -512,6 +512,9 @@ module.exports = function (
                     email: email,
                     errno: err.errno
                   })
+                  if (customsErr) {
+                    throw customsErr
+                  }
```

Comment:
I didn't want to swallow more serious errors, like a DB error. Only when everything succeeded, and there was no account.